hermeto/core/package_managers/gomod.py

```diff
@@ -1255,6 +1255,7 @@ def from_repo_path(cls, repo_path: RootedPath) -> "Self":
         try:
             repo.remote().fetch(force=True, tags=True)
         except Exception as ex:
+            log.error(ex)
```

Comment:
nitpick: This will look weird in the user log, almost as if we had an uncaught error somewhere. I think this should be appended to the `FetchError` string, after all `Failed to fetch the tags on the Git repository` truly begs for a more detailed error message :).

Reply:
That is a good point. I don't think we need to print the stack trace at all. Updated.